GarbageCan.Web/Models/Booster.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace GarbageCan.Web.Models
+{
+    public class Booster
+    {
+        [SwaggerSchema("The XP multiplier. Internally 1.0 is added to the total (e.g. 0.5 -> 1.5)")]
```

Comment:
this is out of date, i decided not to do this because it was dumb

Reply:
Lol fair enough